Marvel/Marvel/HeroesDetails.swift

```diff
@@ -18,4 +18,16 @@ class HeroesDetails: UIViewController {
         dismiss(animated: true, completion: nil)
     }
     
+    var idChar: String!
```

Comment:
Bah, esses force unwrap aqui me incomodam.

Reply:
Me sinto quase pessoalmente agredido

Reply:
Boa, as é um hábito que to revendo toda  vez já.